include/swift/AST/DeclContext.h

```diff
@@ -263,7 +263,10 @@ class alignas(1 << DeclContextAlignInBits) DeclContext {
 
   /// If this DeclContext is a protocol extension, return the extended protocol.
   ProtocolDecl *getAsProtocolExtensionContext() const;
-
+ 
+  /// Retrieve the nearest enclosing nominal type context.
+  NominalTypeDecl *getEnclosingNominalContext() const;
```

Comment:
Nitpick: I don't think it's correct to say a nominal is "enclosing" when you're in an extension.